pimd/pim_cmd.c

```diff
@@ -3701,9 +2499,9 @@ DEFUN (show_ip_pim_assert_winner_metric,
 	return CMD_SUCCESS;
 }
 
-DEFUN (show_ip_pim_interface,
+DEFPY (show_ip_pim_interface,
        show_ip_pim_interface_cmd,
-       "show ip pim [mlag] [vrf NAME] interface [detail|WORD] [json]",
+       "show ip pim [mlag$mlag] [vrf NAME] interface [detail$detail|WORD$word] [json$json]",
```

Comment:
Same with the interface name again here.

btw: it worked before because for `NAME` and `WORD`, if the previous token is a constant (`interface` here), that is assigned as the name for the parameter. So if there is no `$anything` it is the same as `$interface`.

Reply:
Resolved, thank you